packages/template-retail-react-app/app/components/order-summary/index.jsx

```diff
@@ -92,7 +112,7 @@ const OrderSummary = ({
     isEstimate = false,
     fontSize = 'md'
 }) => {
-    basket = basket || useCurrentBasket().basket
```

Comment:
Nit, this line is weird, why do we need this? 🤔 maybe we could just remove it since basket is a prop.

Reply:
Agreed, let pass basket as props and removed the hook use